mtwitter.js

```diff
@@ -0,0 +1,244 @@
+var Twitter = require('twitter');
+var Request = require('request');
+var fs = require('fs');
+var Buffer = require('buffer').Buffer;
+
+var TWITTER_PROFILE_INTERVAL = 60000;
+
+var mtwitter = function(bridge, config) {
+    this.app_auth = config.app_auth;
+    this.app_twitter = null;
+    this.tuser_cache = {};
+    this.timeline_list = [];
+    this.timeline_queue = [];
+    this.timeline_period = 0;
+    this.timeline_intervalobj = null;
+    this._bridge = bridge;
+    this.get_bearer_token = function(cb) {
```

Comment:
I would prefer it if your object functions were defined on the prototype rather than the particular instantiation. Whilst it matters very little for this use case as you're not doing inheritance, there are a number of benefits regardless:
 - One less indent ^___^
 - Clearer separation between constructor and instance functions (not sharing the same variable scopes)
 - Clarity that the defined function is *supposed* to be an instance function (due to the presence of `.prototype`), rather than a random function tacked onto an existing object.

In practice, this means changing this:
```javascript
var mtwitter = function(bridge, config) {
  ...
  this.get_bearer_token = function(cb) {
    ...
  }
}
```
into:
```javascript
var mtwitter = function(bridge, config) {
  ...
}

mtwitter.prototype.get_bearer_token = function(cb) {
  ...
}
```